tests/Responses/Expected/checkout/reselect_shipping_response.json

```diff
@@ -0,0 +1,19 @@
+{
+    "id": @integer@,
+    "items": @array@,
```

Comment:
But we can just instead of matcher use raw value. As I wrote before, it will just help developers to check an api structure.